### DISCUSSION TITLE
🛫 Version 1.2.1

### DISCUSSION THREAD

**Discussion by valnoxy** [valnoxy/t480-oc#15] [Announcements]
Hey 👋,
I hope you all have a great day!

## Changes
The following kexts was added:
   - AppleBacklightFixup
   - Display-30e4-521

The following kexts was updated:
   - **AppleALC**: 1.7.2
   - **BrcmPatchRAM**: 2.6.2
   - **WhateverGreen**: 1.5.9

The following config changes was applied:
   - **Rename HECI to IMEI (PAVP)** -> `true` 
   - **Rename MEI to IMEI (PAVP)** -> `true`
   - **AvoidRuntimeDefrag** -> `true`
   - **DiscardHibernateMap** -> `false`
   - **Interface PciRoot(0x0)/Pci(0x1F,0x3)**
   - **Interface PciRoot(0x0)/Pci(0x2,0x0)**
   - **Interface PciRoot(0x0)/Pci(0x14,0x0)**
   - **PointerSupport** -> `true`

The following ACPI changes was applied:
   - **Removed SSDT-PM.aml**
   - **Added SSDT-GPRW.aml**

## Issues assigned to this release
   - #7 
   - #8 

## Contributor
   - @Comet1903 (see [t480-monterey-opencore](https://github.com/Comet1903/t480-monterey-opencore))

<hr /><em>This discussion was created from the release <a href='https://github.com/valnoxy/t480-oc/releases/tag/v1.2.1'>🛫 Version 1.2.1</a>.</em>

**Comment by Mifaxav** [comment]
Dude, this is amazing!

Trackpad is flawless!

Even the clicking and switching windows with the '3 finger swipe' is just amazing!

Thanks @valnoxy @Comet1903 :D

**Reply by zeeall** [reply]
Yeah, trackpad reliability and gestures are on point now. Shout out to valnoxy and Comet for that :)

**Comment by askwakhid** [comment]
Hi,

First, I would say thank you for you EFI Repositories.
I use your T480 EFI to my T480s ( https://github.com/askwakhid/ThinkPad-T480s-macOS-OpenCore )

In my T480s the thunderbolt function is working fine. 
I can use USB C dongle ( USB C to HDMI/PD 65W/USB A/VGA ) to my 22" FULL HD monitor and everythings works fine.

Thank you.
